src/main.c

```diff
@@ -1,6 +1,6 @@
 #include <stdio.h>
-#include "data.h"
-#include "functions.h"
+#include <data.h>
+#include <functions.h>
```

Comment:
The standard is to only use `<>` for standard or global includes and ``""` for user defined includes.
https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename